tests/polynomial/runner/generate_test_cases.py

```diff
@@ -43,15 +43,15 @@ def test_setup(i: int):
 #  - expected
 #  - coefficient_list
 TEST_TEMPLATE = """
-#ideal_{{test_number}} = #polynomial.polynomial<{{ideal}}>
-#ring_{{test_number}} = #polynomial.ring<cmod={{cmod}}, ideal=#ideal_{{test_number}}>
-!poly_ty_{{test_number}} = !polynomial.polynomial<#ring_{{test_number}}>
+#ideal_{{test_number}} = #polynomial.int_polynomial<{{ideal}}>
+#ring_{{test_number}} = #polynomial.ring<coefficientType = i32, coefficientModulus={{cmod}} : {{cmod_type}}, polynomialModulus=#ideal_{{test_number}}>
```

Comment:
Given that the coefficient type of the ring is hard-coded at i32, why do we need the `gen_tensor_op` sign extension/truncation ops? It seems like all the tests use i32, and I noticed that the coefficient types in the tests were updated to i32, so maybe the test configs don't need it?

Reply:
I think it would be future proof for other coefficient types. I should add another test, but baby has arrived so I just want to get this one in :)